### DISCUSSION TITLE
Why isn't the default chart version in the Helm chart repository increasing?

### DISCUSSION THREAD

**Discussion by civilizeddev** [artifacthub/hub#3633] [Q&A] [answered]
From https://github.com/crossplane/crossplane/discussions/5302:

> Until recently, the helm chart has been updating,
>
> but why is the default version fixed at 1.15.0-rc.0.9.ged19cd5d?
>
> https://artifacthub.io/packages/helm/crossplane/crossplane

I'm experiencing this issue, and I'm wondering if it might be a bug in Artifact Hub. Is it possible to check?

Thank you.

**Comment by tegioz** [comment, marked as answer]
Hi @civilizeddev 👋

It's a bug, thanks for reporting it! The database function that checks the precedence of two semver versions isn't following the spec when comparing the pre-release part. We'll look into it 🙂

**Reply by civilizeddev** [reply]
Thank you for your kind answer.

**Reply by tegioz** [reply]
Fix deployed! (#3636)
